docs/en/01-how-to-build/macos-arm64.md

```diff
@@ -0,0 +1,164 @@
+    - [Build SDK and Demo](#build-sdk-and-demo)
+
+## Install Toolchains
+
+- cmake
+
+  ```
+  brew install cmake
+  ```
+
+- clang
+
+  install Xcode or Command Line Tools
+
+  ```
+  xcode-select --install
+  ```
+
+## Install Dependencies
+
+### Install Dependencies for Model Converter
+
+Please refer to [get_started](../get_started.md) to install conda, PyTorch, mmcv.
+
+### Install Dependencies for SDK
+
+You can skip this chapter if you are only interested in the model converter.
+
+<table class="docutils">
+<thead>
+  <tr>
+    <th>NAME </th>
+    <th>INSTALLATION </th>
+  </tr>
+</thead>
+<tbody>
+  <tr>
+    <td>OpenCV<br>(>=3.0) </td>
+    <td>
+<pre><code>
+brew install opencv
+</code></pre>
+    </td>
+</tbody>
+</table>
+
+### Install Inference Engines for MMDeploy
+
+Both MMDeploy's model converter and SDK share the same inference engines.
+
+You can select you interested inference engines and do the installation by following the given commands.
+
+This document focus on Core ML. The installation of ONNX Runtime, ncnn and TorchScript is similar to the linux platform, please refer to the document [linux-x86_64](linux-x86_64.md) for installation.
+
+The TorchScript model is used as the IR in the conversion process of the Core ML model. In order to support the custom operator, libtorch needs to be installed.
+
+<table  class="docutils">
+<thead>
+  <tr>
+    <th>NAME</th>
+    <th>PACKAGE</th>
+    <th>INSTALLATION</th>
+  </tr>
+</thead>
+<tbody>
+  <tr>
+    <td>Core ML</td>
+    <td>coremltools</td>
+    <td>
+<pre><code>
+pip install coremltools
+</code></pre>
+    </td>
+  </tr>
+  <tr>
+  <td>TorchScript</td>
+  <td>libtorch</td>
+  <td>
+  1. Libtorch doesn't provide prebuilt arm library for macOS, so you need to compile it yourself. Please note that the version of libtorch must be consistent with the version of pytorch. <br>
+  2. Take LibTorch 1.8.0 as an example. You can install it like this:
+<pre><code>
+git clone -b v1.8.0 --recursive https://github.com/pytorch/pytorch.git
+cd pytorch
+mkdir build && cd build
+cmake .. \
+    -DCMAKE_BUILD_TYPE=Release \
+    -DPYTHON_EXECUTABLE=`which python` \
+    -DCMAKE_INSTALL_PREFIX=install \
+    -DDISABLE_SVE=ON
+make -j$(nproc) && make install
+export Torch_DIR=$(pwd)/install/share/cmake/Torch
+</code></pre>
+  </td>
+  </tr>
+</tbody>
+</table>
+
+## Build MMDeploy
+
+```bash
+cd /the/root/path/of/MMDeploy
+export MMDEPLOY_DIR=$(pwd)
+```
+
+#### Build Model Converter
+
+- **Core ML**
+
+  Core ML uses torchscript as IR, so you need to compile torchscript custom operators
+
+- **torchscript** custom operators
+
+  ```bash
+  cd ${MMDEPLOY_DIR}
+  mkdir -p build && cd build
+  cmake -DMMDEPLOY_TARGET_BACKENDS=torchscript -DTorch_DIR=${Torch_DIR} ..
+  make -j$(nproc) && make install
+  ```
+
+Please check [cmake build option](cmake_option.md).
+
+#### Install Model Converter
+
+```bash
+cd ${MMDEPLOY_DIR}
+pip install -e .
```

Comment:
Using `pip install -v -e .` is better since it will show the detailed information